src/site/content/en/blog/coop-coep/index.md

```diff
@@ -23,27 +23,42 @@ origin_trial:
 feedback:
   - api
 ---
+{% Banner 'caution', 'body' %}
+
+**Updates**
+
+**October 15th, 2020**: `self.crossOriginIsolated` is available from Chrome 87.
+Reflecting that, `document.domain` is immutable when `self.crossOriginIsolated`
+returns `true`. `performance.measureMemory` ending its origin trial and is
+planned to be enabled by default in Chrome 88. Shared Array Buffer on Android
+Chrome will be available from Chrome 88.
+
+**September 1st, 2020**: COOP Reporting is behind the `#cross-origin-isolated` flag
+in Chrome 86.
+
+{% endBanner %}
```

Comment:
The banner has more top and bottom padding than I'd like but I still think it's more appropriate than the Gotchas aside:

![image](https://user-images.githubusercontent.com/4713486/96033748-5e0dd900-0e15-11eb-8dd7-b17f038d1471.png)

See previous comment: https://github.com/GoogleChrome/web.dev/pull/4042#pullrequestreview-508634911